backend/src/routes/auth.ts

```diff
@@ -1,10 +1,10 @@
 import express from 'express'
-import { verifyApiKey } from '../middlewares/auth'
+import { verifyApiKey, verifyFirebaseToken } from '../middlewares/auth'
 import { login, logout } from '../controllers/auth'
 
 const router = express.Router()
 
-router.post('/login', verifyApiKey, login)
+router.post('/login', verifyApiKey, verifyFirebaseToken, login)
```

Comment:
verifyFirebaseToken is to verify the session-token no the idToken. After we verify the idToken we generate a sessionCookie that is used by the frontend.